From 2990b982d47715f282eb0bb9e7177789f448389d Mon Sep 17 00:00:00 2001
From: Subrata Chatterjee <schatterjee@habana.ai>
Date: Wed, 12 May 2021 19:50:22 +0300
Subject: [PATCH] [SW-39285] gaudi: comms: add sp regs

SP register fields in communication dynamic
registers map struct initialized with corresponding macros.
These regiters are used for irq polling mode.

Change-Id: I43598794696e52dc6d650cebe1041ccf89d9ec1f
Signed-off-by: Subrata Chatterjee <schatterjee@habana.ai>
---

diff --git a/gaudi/asic.c b/gaudi/asic.c
index e3444fa..710344e 100644
--- a/gaudi/asic.c
+++ b/gaudi/asic.c
@@ -400,4 +400,10 @@
 	regs->hw_state = mmHW_STATE;
 	regs->kmd_msg_to_cpu = mmPSOC_GLOBAL_CONF_KMD_MSG_TO_CPU;
 	regs->cpu_cmd_status_to_host = mmCPU_CMD_STATUS_TO_HOST;
+	regs->gic_host_irq_ctrl = mmGIC_HOST_IRQ_CTRL_POLL_REG;
+	regs->gic_tpc_qm_irq_ctrl = mmGIC_TPC_QM_IRQ_CTRL_POLL_REG;
+	regs->gic_mme_qm_irq_ctrl = mmGIC_MME_QM_IRQ_CTRL_POLL_REG;
+	regs->gic_dma_qm_irq_ctrl = mmGIC_DMA_QM_IRQ_CTRL_POLL_REG;
+	regs->gic_nic_qm_irq_ctrl = mmGIC_NIC_QM_IRQ_CTRL_POLL_REG;
+	regs->gic_dma_core_irq_ctrl = mmGIC_DMA_CR_IRQ_CTRL_POLL_REG;
 }
